### DISCUSSION TITLE
不是死水还阴啊？

### DISCUSSION THREAD

**Discussion by Kk-k11** [share121/inter-knot#790] [其他]
加纳

**Comment by fengxia33** [comment]
不阴我吃

**Comment by lixianyunyou** [comment]
不阴我吃辛久奈大雷

**Comment by light814** [comment]
Σ(°Д°;村里人打来了？

**Comment by wu364** [comment]
我天村里人😱

**Comment by 3077777777** [comment]
抱歉，你的句子中含有“死”字，我可以认为你对死水进行了评价叮，已触发关键词🤓我个人觉得波风水门【秽土转生】真的是最需要操作的忍者了。我对操作的定义就是这件事你知道怎么做但是你做不到。在目前的所有忍者中只有死水能让我有这种体验。不说别的，被先手了替身2A插标闪走，这套操作没玩个几百把是真的做不到的，90%的人想做到这套操作都会发现自己2A根本插不出标来，训练营和决斗场的心理压力是不一样的，你在训练营就算能百分百做到2A插标，决斗场也90%的情况是2A插不出来。

**Comment by gaojiaobei** [comment]
不是，死水还阴啊

**Comment by nahida0-o** [comment]
死水还阴啊

**Comment by Qiling-XwX** [comment]
村里人！

**Comment by shuyishaoxiancao** [comment]
我了个波风水门【秽土转生大人】！
